bulk_strike.py

```diff
@@ -380,7 +400,7 @@ def main():
     if options.action == 'get_info':
         get_info(options.host, options.file, options.log)
     elif options.action == 'get_logins':
-        get_logins(options.host, options.file, options.log, options.clean)
+        get_logins(options.host, options.file, options.log, options.clean, options.cmd_exe)
```

Comment:
is this a typo? should the new argument be added to `start_rtr` function instead?

Reply:
Correct, I will update it.